### PR TITLE
New WPJ string

### PR DESCRIPTION
**Pull request by rpdome**

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java [hunk 1]
```diff
@@ -400,6 +400,16 @@ private ErrorStrings() {
...
                     "Please return the device to your administrator to restart the registration of this device.";
 
+    /**
+     * The device is registered with precreate/userless WPJ, and its registration was deleted by the admin.
+     * This is an irrecoverable error, and the admin has to re-prep the device.
+     * */
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_CODE =
+            "userless_device_deleted_on_server";
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE =
+            "This device configuration was changed on the server. " +
```
Comment (shahzaibj) [inline common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java:410]: Is configuration change same as device deleted?

File: common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java [hunk 2]
```diff
@@ -400,6 +400,16 @@ private ErrorStrings() {
...
+                    "Please return the device to your administrator to restart the registration of this device.";
```
Comment (shahzaibj) [inline common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java:411]:
```suggestion
                    "Please return the device to your administrator to re-register this device.";
```

File: common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java (hunk 1 above)
Comment (rpdome) [reply common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java:410]: I guess I could just say that it's deleted.

File: common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java [hunk 3]
```diff
@@ -397,8 +397,18 @@ private ErrorStrings() {
...
+
+    /**
+     * The device is registered with precreate/userless WPJ, and its registration was deleted by the admin.
+     * This is an irrecoverable error, and the admin has to re-prep the device.
+     * */
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_CODE =
+            "userless_device_deleted_on_server";
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE =
+            "This device was deleted from the tenant. " +
+                    "Please return the device to your administrator to re-register this device.";
```
Comment (shahzaibj) [inline common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java:411]: Looks to be the same value as used for the `REGISTERED_SHARED_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE` declared on line 399. Perhaps there could be a private constant called `DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE` and both the error codes for userless and shared device reference that constant?

**Review by shahzaibj — APPROVED** [review]
Left one comment, else LGTM

**Review by hamiltonha — APPROVED** [review]

**Review by iambmelt — APPROVED** [review]

**Pull request merged**